src/icons/badges/__tests__/__snapshots__/MetadataDefaultBadge.test.tsx.snap

```diff
@@ -16,11 +16,11 @@ exports[`icons/badges/MetadataDefaultBadge should correctly render icon with spe
       cy="8"
       fill="none"
       r="7.5"
-      stroke="#767676"
+      stroke="#6f6f6f"
     />
     <path
       d="M5.126 10.545a.57.57 0 0 1-.693.437.603.603 0 0 1-.416-.728l1.143-4.8c.122-.513.772-.62 1.039-.172L8 8.308l1.801-3.026c.267-.449.917-.34 1.039.173l1.143 4.8a.603.603 0 0 1-.416.727.57.57 0 0 1-.693-.437l-.817-3.43-1.572 2.643a.557.557 0 0 1-.97 0L5.943 7.116l-.817 3.43z"
-      fill="#767676"
+      fill="#6f6f6f"
```

Comment:
I assume this is the hex for bdl-gray-65?

Reply:
yup. I updated the description to include that